### PR TITLE
Test uninitialized input in saturating sub

### PR DESCRIPTION
**Pull request by arr00**
## Tests
* Enhanced test coverage for safe subtraction operations with additional edge cases involving undefined inputs to verify saturation behavior

[Review Change Stack](https://app.coderabbit.ai/change-stack/OpenZeppelin/openzeppelin-confidential-contracts/pull/373?utm_source=github_walkthrough&utm_medium=github&utm_campaign=change_stack)

### CODE REVIEW

File: test/utils/FHESafeMath.test.ts [hunk 1]
```diff
@@ -204,7 +208,7 @@ describe('FHESafeMath', function () {
       operation: '-',
     },
   ]) {
-    describe(`saturating ${params.functionSignature.startsWith('saturatingAdd') ? 'add' : 'sub'}`, function () {
+    describe.only(`saturating ${params.functionSignature.startsWith('saturatingAdd') ? 'add' : 'sub'}`, function () {
```
Comment (arr00) [inline test/utils/FHESafeMath.test.ts]:
```suggestion
    describe(`saturating ${params.functionSignature.startsWith('saturatingAdd') ? 'add' : 'sub'}`, function () {
```
